includes/HealthChecks.php

```diff
@@ -0,0 +1,332 @@
+<?php
+
+namespace NewfoldLabs\WP\Module\Performance;
+
+use NewfoldLabs\WP\ModuleLoader\Container;
+
+/**
+ * Add performance health checks.
+ */
+class HealthChecks {
+
+	/**
+	 * Dependency injection container.
+	 *
+	 * @var Container
+	 */
+	protected $container;
+
+	/**
+	 * Constructor.
+	 *
+	 * @param Container $container Dependency injection container.
+	 */
+	public function __construct( Container $container ) {
+		$this->container = $container;
+	}
+
+	/**
+	 * Add health checks.
+	 */
+	public function addHealthChecks() {
+		$manager = $this->container->get( 'healthCheckManager' );
+
+		// PRESS7-108: Autosave Interval.
+		$manager->addHealthCheck(
+			array(
+				'id'    => 'autosave-interval',
+				'title' => __( 'Autosave Interval', 'newfold-performance-module' ),
```

Comment:
```suggestion
				'title' => __( 'Autosave Interval', 'newfold-module-performance' ),
```

I think this is the right namespace we're using for the i18n function.